ui/src/warp_runner/mod.rs

```diff
@@ -156,20 +160,37 @@ impl WarpRunner {
                                         break;
                                       }
                                 }
-                                Err(_e) => {
-                                    // todo: log error
+                                Err(e) => {
+                                    logger::error(&format!("failed to convert raygun event: {}", e));
                                 }
                             }
                         }
                     },
-
                     // receive a command from the UI. call the corresponding function
                     opt = rx.recv() => {
                         //println!("got warp_runner cmd");
                         match opt {
                         Some(cmd) => match cmd {
                             WarpCmd::Tesseract(cmd) => handle_tesseract_cmd(cmd, &mut tesseract).await,
-                            WarpCmd::MultiPass(cmd) => handle_multipass_cmd(cmd, &mut tesseract, &mut account).await,
+                            WarpCmd::MultiPass(cmd) => {
+                                // if a command to block a user comes in, need to update the UI because warp doesn't generate an event for a user being blocked.
+                                // todo: ask for that event
```

Comment:
In https://github.com/Satellite-im/Warp/pull/148 there will be an event in `MultiPassEventKind` that will notify the client of when they blocked or unblocked a user (eg if you were to use `MultiPass::block` it would emit an event to the stream letting you know they were successfully blocked apart from the function returning `Ok`). In a separate PR once that is merged will notify the stream of when they are blocked/unblocked (though still wondering on if i should allow that to be emitted to the stream or not)